frontend/app/framework/angular/pipes/date-time.pipes.spec.ts

```diff
@@ -98,6 +99,16 @@ describe('FromNowPipe', () => {
         expect(actual).toBe(expected);
     });
 
+    it('should format to from now string in Dutch', () => {
+        DateHelper.setlocale('nl');
```

Comment:
I think this is critical, because it can easily brake other tests.

It is better to add an optional parameter to the pipe to configure the locale and then test it.

Reply:
It's critical because i use the `Datehelper.setlocale('nl')`?

Because then we also need to fix this test:
https://github.com/Squidex/squidex/blob/f38492ea177e7e154a8a2c22b721bf826e915fa3/frontend/app/framework/utils/date-helper.spec.ts#L20

Reply:
you should at least reset it after your test with a try ... finally
. The problem is concurrency, but I guess this is not a issue with node

Reply:
Yea you are right about resetting, I will add an finally